packages/icon/src/vaadin-iconset.d.ts

```diff
@@ -30,12 +47,6 @@ declare class Iconset extends ElementMixin(HTMLElement) {
    * over the size defined by the user to ensure correct appearance.
    */
   size: number;
-
-  /**
-   * Produce SVGTemplateResult for the element matching `name` in this
-   * iconset, or `undefined` if there is no matching element.
-   */
-  applyIcon(name: string): { svg: IconSvgLiteral; size: number; viewBox: string | null };
```

Comment:
Removed this method in favor of the new `getIconSvg()` static method to align with other methods.